src/module/api/texupd/create/index.ts

```diff
@@ -0,0 +1,67 @@
+import * as env from 'module/env';
+import {
+  CreateI,
+  CreateI_Obj,
+  CreateI_Obj_Property,
+} from 'module/api/texupd/proto/create_pb';
+import { APIClient } from 'module/api/texupd/proto/ApiServiceClientPb';
+import {INewUpdate} from 'module/interface/update'
+
+const AUDIENCEIDKEY = 'audience.venturemark.co/id';
+const ORGANIZATIONIDKEY = 'organization.venturemark.co/id';
+const TIMELINEIDKEY = "timeline.venturemark.co/id";
+const USERIDKEY = 'user.venturemark.co/id';
```

Comment:
It would be cool if we could put these magic strings into one common place so that we can use shared code and not duplicate it all over the place.